### PR TITLE
[TEVA-1684] fix notification bg color/spacing/layout for all pages Part 1

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-1684

There is more work to do to decouple dashboard styles from 'mast' area styles but this will be done in subsequent PR

### CODE REVIEW

File: app/frontend/src/styles/application/dashboard.scss [hunk 1]
```diff
@@ -7,8 +7,7 @@
   width: 100vw;
 
   > .govuk-width-container {
-    background-color: govuk-colour('white');
-    padding-top: govuk-spacing(7);
+    background-color: inherit;
```
Comment (cpjmcquillan) [inline app/frontend/src/styles/application/dashboard.scss:10]: nice 👍

File: app/views/publishers/organisations/schools/edit.html.haml [hunk 2]
```diff
@@ -1,4 +1,6 @@
-= content_for :page_title_prefix, @organisation.name.titlecase
+.govuk-grid-row{ class: 'govuk-!-margin-top-7' }
```
Comment (cpjmcquillan) [inline app/views/publishers/organisations/schools/edit.html.haml:1]: I'd imagine this should actually just be
```
- content_for :page_title_prefix, @organisation.name.titlecase
```

File: app/views/publishers/organisations/schools/index.html.haml [hunk 3]
```diff
@@ -1,4 +1,6 @@
-= content_for :page_title_prefix, current_organisation.name
+.govuk-grid-row{ class: 'govuk-!-margin-top-7' }
```
Comment (cpjmcquillan) [inline app/views/publishers/organisations/schools/index.html.haml:1]: Same as above

**Comment by david-mears-dfe** [comment]
Why do some pages have margin-top-7 and others have margin-top-4?

**Comment by alexbowen** [comment]
> Why do some pages have margin-top-7 and others have margin-top-4? I'm sure there's a reason but perhaps it could be made clearer either in the commit message or with some clever class naming.

sorted this out, all done in css rather than repetitious classes being added

**Review by csutter — APPROVED** [review]

**Pull request merged**